src/Components/Status/Status.ts

```diff
@@ -0,0 +1,5 @@
+export const Status = {
```

Comment:
it's not a component - it's a type, so should live in `type` (types) folder
also nesting inside folder is redundant